### PR TITLE
sphinxjp.themecore: initial add

### PR DESCRIPTION
**Pull request by johannesring**

### CODE REVIEW

File: recipes/sphinxjp.themecore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - sphinx
+
+test:
+  imports:
+    - sphinxjp
+    - sphinxjp.themecore
+
+about:
+  home: http://bitbucket.org/shimizukawa/sphinxjp.themecore
+  license: MIT License
```
Comment (jakirkham) [inline recipes/sphinxjp.themecore/meta.yaml]: `MIT` is sufficient. Please drop the rest.

Comment (jakirkham) [inline recipes/sphinxjp.themecore/meta.yaml]: Appears upstream does not include a license file. Please note that in a comment here and note that it should be added once there is one (see the [example]( https://github.com/conda-forge/staged-recipes/blob/3b962af8c4da15da4cb8386763538d6065e6c5fc/recipes/example/meta.yaml#L46-L49 )).

Also please raise an issue with upstream asking for a license file to be added to the repo and `MANIFEST.in`. Please make sure to clarify that the MIT license requires us to include it in all distributions. Hence why they really need to provide one so we can comply with their licensing terms. Please cross reference this PR in that issue and that issue in this PR once created.

**Comment by johannesring** [comment]
@jakirkham - I have filed an issue about the license file [here](https://bitbucket.org/shimizukawa/sphinxjp.themecore/issues/2/missing-license-text).

File: recipes/sphinxjp.themecore/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docutils
+    - sphinx
```
Comment (jakirkham) [inline recipes/sphinxjp.themecore/meta.yaml]: We should be able to drop `docutils` and `sphinx` from `build`. However, that doesn't need to be dealt with here.

**Comment by jakirkham** [comment]
Thanks @johannesring.

**Comment by jakirkham** [comment]
You should get an email that invite you to join conda-forge, @johannesring. Once accepted you will be added to a team with the same name as this recipe. Those will give you permissions on the feedstock (repo) for this recipe.

Make sure when proposing any change that you go through the typical GitHub workflow of forking the feedstock and making changes in your fork that you PR back. Once merged CIs will build and deploy any changes you make. Please let us know if you have any questions and welcome to conda-forge.

**Comment by johannesring** [comment]
Great! Thanks @jakirkham.

**Pull request merged**